### ISSUE TITLE
NoClassDefFoundError for SBT 0.13.8

### ISSUE THREAD

**Issue by d6y** [orrsella/sbt-sublime#11]
Hello - and thanks for this super-useful plugin.

I've noticed a problem when running `gen-sublime` using SBT 0.13.8:

```
[info] Writing project to file: /Users/richard/tmp/sublime-scalap/sublime-scalap.sublime-project
java.lang.NoClassDefFoundError: scala/tools/scalap/scalax/rules/scalasig/ScalaSigSymbol
```
I've put full details in a minimal project: https://github.com/d6y/sublime-scalap

The work-around is to use 0.13.7 instead.

**Comment by orrsella** [comment]
Thanks for bringing this to my attention and for taking the time to add all the details.

I just released version `1.0.11` of the plugin which fixes this issue. It seemed to be around a problem with `json4s` which used some reflection classes that depended on `scalac`. That doesn't seem like a good idea to me, and I switched to using [spray-json](https://github.com/spray/spray-json) instead.

Please give the new version a spin and let me know if you still have any problems:

```scala
addSbtPlugin("com.orrsella" % "sbt-sublime" % "1.0.11")
```

If it's fine please go ahead and close the issue.

Thanks!

**Comment by d6y** [comment]
Version 1.0.11 is indeed fine - thanks for such a quick fix for this!
